app/src/main/java/de/grobox/transportr/utils/DateUtils.kt

```diff
@@ -71,7 +73,7 @@ object DateUtils {
         val delayMinutes = millisToMinutes(delay)
         return Delay(
             delay = "${if (delayMinutes >= 0) '+' else ""}$delayMinutes",
-            color = ContextCompat.getColor(context, if (delayMinutes > 0) R.color.md_red_500 else R.color.md_green_500)
+            color = MaterialColors.getColor(context, if (delayMinutes > 0) R.attr.colorError else R.attr.colorPrimary, Color.TRANSPARENT)
```

Comment:
Not sure if the Material You colors are meant to be used as text color?

Reply:
Yes, see "Primary" roles at https://m3.material.io/styles/color/roles